### PR TITLE
feat: migrate old token list to new

### PR DESCRIPTION
**Pull request by NikoManiac**
closes #

### CODE REVIEW

File: packages/web3-shared/src/context/index.tsx [hunk 1]
```diff
@@ -8,7 +8,6 @@ export type { Web3ProviderType } from './type'
 
 export const Web3ProviderContext = createContext<Web3ProviderType>(null!)
 
-/** @internal */
 export function useWeb3Context() {
```
Comment (Jack-Works) [inline packages/web3-shared/src/context/index.tsx:11]: I think it's better to not expose the context directly.

File: packages/shared/src/UI/components/ERC20TokenList/ERC20TokenListItem.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+                            <span>{formatBalance(data.balance, token.decimals)}</span>
+                        ) : (
+                            <MaskLoadingButton
+                                variant="rounded"
+                                color="primary"
+                                onClick={onImport}
+                                size="small"
+                                clearStyle
+                                loadingIndicator={<LoadingIcon sx={{ fontSize: 16 }} />}>
+                                Import
```
Comment (guanbinrui) [inline packages/shared/src/UI/components/ERC20TokenList/ERC20TokenListItem.tsx:125]: i18n

File: packages/theme/src/Components/LoadingButton/index.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+    const handleClick = useCallback(
+        async (event: React.MouseEvent<HTMLButtonElement>) => {
+            setLoading(true)
+            try {
+                await onClick(event)
+                setLoading(false)
+            } catch (error) {
+                setLoading(false)
+                throw new Error(error as string)
```
Comment (septs) [inline packages/theme/src/Components/LoadingButton/index.tsx:22]:
```suggestion
                if (error instanceof String) {
                    throw new Error(error)
                }
```

Comment (NikoManiac) [reply packages/theme/src/Components/LoadingButton/index.tsx]:
```
if (typeof error === 'string') {
    throw new Error(error)
}
```

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**